### ISSUE TITLE
Problem with opkg_dist_feeds

### ISSUE THREAD

**Issue by NN708** [ganto/ansible-openwrt#20]
Hello ganto,

I'm trying to use this collection on OpenWrt. In the `opkg` role, `opkg_dist_feeds` has a default value. However, OpenWrt distributions already have a default `/etc/opkg/distfeeds.conf` file, and its content varies according to version and architecture. For example, OpenWrt 19.07 has a `openwrt_freifunk` feed which OpenWrt 18.06 and 21.02 don’t have, and different architectures have different URLs for `openwrt_kmods` feed (https://downloads.openwrt.org/releases/19.07.7/targets/armvirt/32/kmods/4.14.221-1-fd72bb7d9bf98eabfd6d62b0918ee30e for armvirt32 and https://downloads.openwrt.org/releases/19.07.7/targets/x86/64/kmods/4.14.221-1-68143adfcb7fc62a239c4be112fe40de for x86_64).

I don't know weather TurrisOS is the same, if this is the case, maybe not changing `/etc/opkg/distfeeds.conf` file by default is better?

**Comment by ganto** [comment]
Yes, I became aware of this recently when I tried to see how well this collection would work on a vanilla OpenWRT on a Raspberry Pi. I realize that it might be quite tricky to generate the file correctly for all architectures and even more when they have different repositories for different releases. Initially, I only tested this in a LXC container where all the kmod packages (and feeds) were missing entirely.

As you suggest I might just completely drop the configuration of this file (or make it optional). Thanks for reminding me. I'll fix this soon.
